### ISSUE TITLE
Blue print for ember-orbit is not converting files into type script after the addon was added

### ISSUE THREAD

**Issue by sarangsbabu367** [orbitjs/ember-orbit#231]
ember orbit blue print is creating files in the format .js after type script addon was added. But generated controllers, routes, etc are created as .ts files.

**Comment by tchak** [comment]
It is true it would be nice to have `TypeScript` versions of blueprints. Another problem you will run into with `TypeScript` is that `store` methods can't be properly typed right now.

**Comment by sarangsbabu367** [comment]
Yes, my typescript code is showing errors like, "store has no method"...

**Comment by dgeb** [comment]
I believe that the general typing issues have been solved with the `Store` and `Cache`. However, we do still need to figure out how to selectively use TS blueprints. It's an annoying paper cut to have to convert these manually.

**Comment by dgeb** [comment]
Relevant [Ember RFC](https://github.com/emberjs/rfcs/pull/776) that addresses this very issue.
